models/training-tuning-scripts/root-cause-models/requirements.txt

```diff
@@ -0,0 +1,8 @@
+cudf==22.8.1
+cupy==10.6.0
+numpy==1.22.4
+pandas==1.3.5
+scikit_learn==1.1.3
+torch==1.12.0+cu113
```

Comment:
torch was updated here to fix issue in sequence classifier (used in phishing and root cause). was getting cuda error with larger batch sizes. fixed in torch 1.12. morpheus torch should also be updated at some point.

Reply:
Good catch. Probably too risky to update that now. Can you make this into an issue?

Reply:
Resolving for release